### PR TITLE
fix: don't use deprecated currentlyFocusedField

### LINKED ISSUE #8457: currentlyFocusedField is deprecated and will be removed in a future release. Use currentlyFocusedInput
**Current Behavior**

I am using a compound navigation setup. It goes `NavigationContainer > AuthStack`. AuthStack consists of `TabStack` as well as a few screens. If I am in one of those screens and swipe back (on iOS, native gesture), I get this warning: 
````
currentlyFocusedField is deprecated and will be removed in a future release. Use currentlyFocusedInput
````

I think it might be caused by the RN release candidate, so if it is, this would be a good warning to address. This seems similar to #8414, but I don't know if they're related.

**Environment**

| software                       | version |
| ------------------------------ | ------- |
| iOS                 |13.6 latest beta|
| @react-navigation/native       |^5.5.1|
| @react-navigation/stack        |^5.5.1|
| react-native-gesture-handler   |^1.6.1|
| react-native-safe-area-context |^3.0.5|
| react-native-screens           |^2.8.0|
| react-native                   |0.63.0-rc.1|
| expo                           |N/A|
| node                           |14.4.0|
| yarn                    |2.0.0-rc.33|

### PR DESCRIPTION
**Pull request by watadarkstar**
currentlyFocusedField => currentlyFocusedInput

Fixes #8457 for 4.x because `createKeyboardAwareNavigator.js` is not inside the `main` branch.

Fixed in main here: https://github.com/react-navigation/react-navigation/commit/35d6b9e3a4a28a59b3b11a67acbf7753d41705ae

### CODE REVIEW

**Comment by watadarkstar** [comment]
@satya164 Please review when you have time :)

**Comment by sheydarezaee** [comment]
any update on this one please? 🙋🏻‍♀️

**Comment by raul-madrigal** [comment]
Would be great to have this one merged ASAP!!

**Comment by sheydarezaee** [comment]
Yes please, our new release version is blocked by this, I have 4.x version and need the patch! 🙏🏻

**Comment by hunterlong** [comment]
+1

**Comment by watadarkstar** [comment]
@sheydarezaee You can patch it yourself with https://www.npmjs.com/package/patch-package 

If its really blocking you until @satya164 can review, merge and release this fix :)

**Comment by Pointbreak-dev** [comment]
+1

**Comment by mhashim6** [comment]
that would be great.

**Comment by JerakRus** [comment]
@satya164 привет, можете ли вы рассмотреть изменения как можно быстрее ?

**Comment by watadarkstar** [comment]
@JerakRus Satyajit is taking some time off. I spoke to him on twitter. He will be back next week.

For now, you can can patch it yourself with https://www.npmjs.com/package/patch-package

**Review by big-toni — APPROVED** [review]

**Comment by cokealmonacid** [comment]
Any news on this?

**Review by satya164 — APPROVED** [review]
Thanks!

**Comment by ebaynaud** [comment]
@satya164 is it possible to release that in something like a 4.4.1?

**Comment by talhafayyaz12** [comment]
I think it is released. Any news on this.

**Comment by JulioOrellana** [comment]
How can I patch this myself? I did use patch-package from `npx patch-package react-navigation/@react-navigation/native` or `yarn patch-package react-navigation` but none detects changes.

**Comment by ebaynaud** [comment]
@JulioOrellana see https://github.com/react-navigation/react-navigation/issues/8457#issuecomment-661445065

**Comment by ebaynaud** [comment]
@JulioOrellana `npx patch-package @react-navigation/native` maybe?
I also had to edit files in lib folder in addition to src before creating the patch...

**Comment by Aure77** [comment]
Any plan to release it in 4.x ?

**Comment by KishoreBarik** [comment]
> 
> 
> I think it is released. Any news on this.

It's not yet released.
image

**Comment by big-toni** [comment]
Why they do not release this?

**Comment by dariusbiro** [comment]
Guys, cmon. Please release this...

**Comment by DigitalDevigner** [comment]
If anybody finds this error annoying right now I use:

`LogBox.ignoreLogs(['currentlyFocusedField']);`

This way it doesn't block other legit errors.  Once this fix is released I will remove this.

**Comment by fdagostino** [comment]
+1

**Comment by thanhcuong1990** [comment]
Please release this!

**Comment by veleda3** [comment]
waiting for this release to update my LogBox

> If anybody finds this error annoying right now I use:
> 
> `LogBox.ignoreLogs(['currentlyFocusedField']);`
> 
> This way it doesn't block other legit errors. Once this fix is released I will remove this.

**Comment by karanmartian** [comment]
Hey facing same issue RNav 4.x. Sincere request to fix it in urgency.

**Comment by jsbls** [comment]
🚀 4.4.1 was released earlier today on [npmjs](https://www.npmjs.com/package/react-navigation). There are also releases for stack, tabs, etc.

**Comment by karanmartian** [comment]
Yes updated already but it still throws same warning. 4.4.1 did not help.

**Comment by jsbls** [comment]
@karanmartian I got a different error, but it seems to be related to `react-native-router-flux` rather than this. Did you update all dependencies? (navigation, stack, tabs, etc.) Maybe removing and reinstalling `node_modules`?

**Comment by karanmartian** [comment]
Yea I updated all. Also updated to Expo 39 sdk. will try reinstalling nm. but that shud not be a solution

**Comment by erbud** [comment]
I have upgraded the `react-navigation` package from version `4.0.10` to version `4.4.1` and the warnings have disappeared.

I am left wondering if I should also update the packages…

```
"react-navigation-drawer": "^ 2.2.2",
"react-navigation-header-buttons": "^ 2.1.2",
"react-navigation-redux-debouncer": "0.0.2",
"react-navigation-stack": "^ 1.9.4",
"react-navigation-tabs": "^ 2.5.6"
```

…to a specific version.

**Comment by karanmartian** [comment]
I am still having the same issue. This hasn't been fixed with 4.4.1. I am on latest Expo 39 SDK and my dependencies are below. What I am doing wrong?

"dependencies": {
        "@expo/vector-icons": "^10.2.1",
        "@react-native-community/async-storage": "~1.12.0",
        "@react-native-community/datetimepicker": "3.0.0",
        "@react-native-community/masked-view": "0.1.10",
        "@types/react": "~16.9.35",
        "@types/react-native": "~0.63.2",
        "add": "^2.0.6",
        "crypto-js": "3.3.0",
        "dateformat": "^3.0.3",
        "expo": "^39.0.0",
        "expo-asset": "~8.2.0",
        "expo-av": "~8.6.0",
        "expo-blur": "~8.2.0",
        "expo-camera": "~9.0.0",
        "expo-constants": "~9.2.0",
        "expo-file-system": "~9.2.0",
        "expo-font": "~8.3.0",
        "expo-image-manipulator": "~8.3.0",
        "expo-image-picker": "~9.1.0",
        "expo-linear-gradient": "~8.3.0",
        "expo-permissions": "~9.3.0",
        "expo-splash-screen": "~0.6.1",
        "expo-status-bar": "~1.0.2",
        "jwt-decode": "^2.2.0",
        "moment": "^2.29.0",
        "prop-types": "^15.7.2",
        "react": "16.13.1",
        "react-native": "https://github.com/expo/react-native/archive/sdk-39.0.0.tar.gz",
        "react-native-appearance": "~0.3.3",
        "react-native-dropdown-picker": "^3.1.11",
        "react-native-elements": "^1.2.7",
        "react-native-gesture-handler": "~1.7.0",
        "react-native-hyperlink": "^0.0.19",
        "react-native-icon-badge": "^1.1.3",
        "react-native-keyboard-aware-scroll-view": "^0.9.2",
        "react-native-keyboard-spacer": "^0.4.1",
        "react-native-modal": "^11.5.6",
        "react-native-modal-datetime-picker": "^8.9.3",
        "react-native-phone-input": "^0.2.4",
        "react-native-reanimated": "~1.13.0",
        "react-native-safe-area-context": "3.1.4",
        "react-native-scalable-image": "^1.0.0",
        "react-native-screens": "~2.10.1",
        "react-native-search-header": "^0.3.5",
        "react-native-svg": "12.1.0",
        "react-native-thumbnail-video": "^0.1.2",
        "react-native-vector-icons": "^7.1.0",
        "react-navigation": "^4.4.1",
        "react-navigation-stack": "^2.8.3",
        "react-navigation-tabs": "^2.9.1",
        "react-nl2br": "^0.6.0",
        "react-redux": "^7.2.1",
        "react-refresh": "^0.8.3",
        "redux": "^4.0.5",
        "slug": "^3.3.4",
        "uuid": "^3.4.0"
    },
    "resolutions": {
        "crypto-js": "3.3.0"
    }

**Comment by Brotakuu** [comment]
@karanmartian my error went away after updating react navigation packages and running `expo install react-native-gesture-handler react-native-reanimated react-native-screens react-native-safe-area-context @react-native-community/masked-view`

**Comment by brianmharding13** [comment]
@karanmartian Were you able to resolve your issue? I'm having the same problem.

**Comment by karanmartian** [comment]
I did something which wasn't exactly a fix. Don't remember exactly though with my huge codebase:(

**Pull request merged**
